packages/website/src/building-blocs/Demo.tsx

```diff
@@ -47,19 +48,28 @@ const useStyles = createStyles((theme, {grow, noPadding}: DemoComponentProps) =>
     preview: {
         backgroundColor: 'white',
         minHeight: 100,
+        display: 'flex',
+        flexDirection: 'column',
     },
     previewWrapper: {
         padding: noPadding ? 0 : theme.spacing.md,
         height: grow ? MAX_HEIGHT : '100%',
     },
+    flexPreviewWrapper: {
+        padding: noPadding ? 0 : theme.spacing.md,
+        display: 'flex',
+        flexDirection: 'column',
+        minHeight: 0,
+        flex: 1,
+    },
     code: {
         minHeight: 100,
         position: 'relative',
         backgroundColor: '#1E1E1E',
     },
 }));
 
-const Demo = ({children, snippet, center = false, grow = false, title, layout, noPadding}: DemoProps) => {
+const Demo = ({children, snippet, center = false, grow = false, title, layout, noPadding, maxHeight}: DemoProps) => {
```

Comment:
I don't mind having this new prop on demo components, but just make sure the code side obey the same height


https://github.com/coveo/plasma/assets/35579930/fa92a18c-25c3-45a1-8266-f91d2b7d5ff4

Reply:
Fixed it. ✅